t/keyval/keyvalemptyvalue.tex

```diff
@@ -0,0 +1,10 @@
+\documentclass{article}
+
+\usepackage{keyval}
+\makeatletter
+\define@key{createvalue}{empty}{\def\myempty{#1}}
+\setkeys{createvalue}{empty={}}
```

Comment:
I suggest adding `{{}}` (and maybe `{{{}}}` too), to be sure that the *correct* number of braces is stripped out. When I reported the issue, I fixed it locally by stripping all braces, but that was wrong too.

Reply:
Great point, the check only comes back empty for `{}` and `{{}}` in keyval.

The stripBraces fix is correct for the intended semantics of "stripBraces", but it looks like keyval should only strip them twice at most. Maybe I can add a "max_layers" parameter here...

Reply:
Alright, a little scary but I updated the test to request exactly 2 braced strips for this case, extending `stripBraces` to allow for a number of layers.